src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -4,30 +4,20 @@
 // 1. If not already done, enable the Google Cloud Logging API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/logging
-// 2. This sample uses Application Default Credentials for authentication.
+// 2. Install the Node.js client library by running
+//    `npm install googleapis --save`
+// 3. This sample uses Application Default Credentials for authentication.
```

Comment:
yup haven't gotten around to working on this again - i'll make sure to apply the same feedback given in the java reviews